pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2425405487>#9833e1</a><span> - 10/21/2024, 07:37 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The configuration path is hardcoded, reducing flexibility and portability across different environments.
</div>
<div id="fix">
<b>Fix:</b> Use environment variables or command-line arguments to specify the configuration path, making the application more flexible and adhering to the Twelve-Factor App methodology.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -30,1 +30,5 @@
 -		viper.AddConfigPath("/root")
 +		configPath := os.Getenv("CONFIG_PATH")
 +		if configPath == "" {
 +			configPath = "/root" // Default path
 +		}
 +		viper.AddConfigPath(configPath)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged